sky/templates/nebius-ray.yml.j2

```
@@ -0,0 +1,72 @@
+cluster_name: {{cluster_name_on_cloud}}
+
+# The maximum number of workers nodes to launch in addition to the head node.
+max_workers: {{num_nodes - 1}}
+upscaling_speed: {{num_nodes - 1}}
+idle_timeout_minutes: 60
+
+provider:
+  type: external
+  module: sky.skylet.providers.nebius.NebiusNodeProvider
```

```suggestion
  module: sky.provision.nebius
```
i think this should be the provision module? as we are using the new provisioner here